sdk/src/main/java/io/dapr/actors/runtime/Remindable.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.actors.runtime;
+
+/**
+ * TODO
+ */
+public interface Remindable {

```

Comment:
It should have a RecieveReminder method, is it TODO?

Reply:
TODO because we are not implementing this yet. It is just to make the code compile and run for the code we cover in this PR. More to come.